homeassistant/components/system_bridge/media_source.py

```diff
@@ -0,0 +1,136 @@
+"""System Bridge Media Source Implementation."""
+from __future__ import annotations
+
+from systembridge import Bridge
+from systembridge.objects.filesystem.file import FilesystemFile
+
+from homeassistant.components.media_player.const import MEDIA_CLASS_DIRECTORY
+from homeassistant.components.media_source.const import (
+    MEDIA_CLASS_MAP,
+    MEDIA_MIME_TYPES,
+)
+from homeassistant.components.media_source.models import (
+    BrowseMediaSource,
+    MediaSource,
+    MediaSourceItem,
+    PlayMedia,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import SystemBridgeDataUpdateCoordinator
+
+ROOT_PATHS: list[str] = [
```

Comment:
Nit: If `ROOT_PATHS` is never to be modified, I would prefer an immutable tuple instead.